src/EventService/Startup.cs

```diff
@@ -0,0 +1,158 @@
+using EventService.Data.Provider.MsSql.Ef;
+using HealthChecks.UI.Client;
+using LT.DigitalOffice.EventService.Models.Dto.Configurations;
+using LT.DigitalOffice.Kernel.BrokerSupport.Configurations;
+using LT.DigitalOffice.Kernel.BrokerSupport.Extensions;
+using LT.DigitalOffice.Kernel.BrokerSupport.Helpers;
+using LT.DigitalOffice.Kernel.BrokerSupport.Middlewares.Token;
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Kernel.EFSupport.Extensions;
+using LT.DigitalOffice.Kernel.EFSupport.Helpers;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Middlewares.ApiInformation;
+using MassTransit;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace EventService
+{
+  public class Startup : BaseApiInfo
+  {
+    public const string CorsPolicyName = "LtDoCorsPolicy";
+    public IConfiguration Configuration { get; }
+
+    private readonly BaseServiceInfoConfig _serviceInfoConfig;
+    private readonly RabbitMqConfig _rabbitMqConfig;
+
+    public Startup(IConfiguration configuration)
```

Comment:
please, fix the order:
private methods
constructor
public methods

Reply:
Replaced methods